catalogue/webapp/pages/works.tsx

```diff
@@ -57,6 +58,8 @@ const Works: NextPage<Props> = ({
   const [loading, setLoading] = useState(false);
   const [, setSavedSearchState] = useSavedSearchState(worksRouteProps);
 
+  useHotjar(Boolean(works && works.results.length > 0));
```

Comment:
I am wondering for this test does it need to trigger the name ? as the results page is not a specific page...

Reply:
Don't think so – the test is for all search result pages, which can be determined by the URL (i.e. anything that starts `/works?query=`). As I understood it, the reason you needed the trigger for the individual archive pages was because there was no way of knowing that it was an archive item from the URL – as far as the URL was concerned, it was just any other work page.

Reply:
Yeah that was correct. this should work as expected as written .